src/system/Nav/NavItem.tsx

```diff
@@ -0,0 +1,133 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import React, { Ref, forwardRef } from 'react';
+import * as NavigationMenu from '@radix-ui/react-navigation-menu';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { VIP_NAV } from '.';
+import { Theme, ThemeUIStyleObject } from 'theme-ui';
+import { NavProps } from './Nav';
+
+interface NavItemTheme extends Theme {
+	outline?: Record< string, string >;
+}
+
+export interface NavItemBaseProps extends NavigationMenu.NavigationMenuItemProps {
+	className?: string;
+	sx?: ThemeUIStyleObject | undefined;
+}
+
+const NavItemBase = forwardRef< HTMLLIElement, NavItemBaseProps >(
+	( { children, sx = {}, className, ...rest }: NavItemBaseProps, ref: Ref< HTMLLIElement > ) => (
+		<NavigationMenu.Item
+			className={ classNames( `${ VIP_NAV }-item`, className ) }
```

Comment:
Dumb question, why do we accept a `className` prop on these components? Don't we want to block additional styling?

Reply:
@chriszarate All components do that. Usually, this is useful for @simonwheatley  to make his Pendo magic, etc. It also gives flexibility to the tool.

Reply:
Got it, but just thinking from a design governance perspective, what are the valid use cases if you remove the Pendo example (which could I think be implemented with `id` or `data-` attribute)?

Reply:
To allow id or data, you would have to allow ...props. In this case, classes is a way to go for simple alternatives. We started to include classes mainly for tracking purposes; that's it. There's no other reason, and we already have tracking set up by Simon with classes.